Scripts/release.sh

```diff
@@ -0,0 +1,276 @@
+#!/bin/bash
```

Comment:
i removed that because it was stopping when an attempt to tag is made and the tag already exists. i need the script to continue after this because someone can be re-running the script to get somewhere after the tag, like the tag push. i think there's enough stop points in the script to not have too much failing activity continue. also, the user can stop the script manually.